mutation/atomic_cell.cc

```diff
@@ -194,14 +194,13 @@ operator<<(std::ostream& os, const atomic_cell_view::printer& acvp) {
         std::ostringstream cell_value_string_builder;
         if (type.is_counter()) {
             if (acv.is_counter_update()) {
-                cell_value_string_builder << "counter_update_value=" << acv.counter_update_value();
+                fmt::print(os, "counter_update_value={}", acv.counter_update_value());
```

Comment:
If the intention is to write directly into os and not even use cell_value_string_builder, you should probably get rid of cell_value_string_builder and not keep it an empty relic.

Reply:
yeah, the general intention is to use `fmt::join()` for better performance and more readable code, and remove `to_string()` helpers in longer run. but i want to make do this in an iterative way -- just focus on one thing if possible.

Reply:
So I don't understand, is cell_value_string_builder still used in this code after you stopped using it in those lines you modified here?

Reply:
Good.
This is exactly what scares me every time I see one of these code-cleanup patches: Among 100 innocent changes there may be one bug hiding, and I don't trust our test suite enough to trust that it will catch any regression. Especially not in the area of error messages (where tests often, rightly, don't insist on the exact text of the error message).